msg/versioned/AirspeedValidated.msg

```diff
@@ -1,22 +1,28 @@
+# Validated airspeed
+#
+# Provides information about airspeed (indicated, true, calibrated) and the source of the data.
+# Used by many services and published to MAVLink.
+
+
 uint32 MESSAGE_VERSION = 1
 
-uint64 timestamp				# time since system start (microseconds)
+uint64 timestamp  # [us] Time since system start
 
-float32 indicated_airspeed_m_s			# [m/s] Indicated airspeed (IAS), set to NAN if invalid
-float32 calibrated_airspeed_m_s     		# [m/s] Calibrated airspeed (CAS), set to NAN if invalid
-float32 true_airspeed_m_s			# [m/s] True airspeed (TAS), set to NAN if invalid
+float32 indicated_airspeed_m_s   # [m/s] [@invalid NaN] Indicated airspeed (IAS)
+float32 calibrated_airspeed_m_s  # [m/s] [@invalid NaN] Calibrated airspeed (CAS)
+float32 true_airspeed_m_s        # [m/s] [@invalid NaN] True airspeed (TAS)
 
-int8 airspeed_source				# Source of currently published airspeed values
-int8 DISABLED = -1
-int8 GROUND_MINUS_WIND = 0
-int8 SENSOR_1 = 1
-int8 SENSOR_2 = 2
-int8 SENSOR_3 = 3
-int8 SYNTHETIC = 4
+int8 airspeed_source        # Source of currently published airspeed values
+int8 DISABLED = -1          # Disabled
+int8 GROUND_MINUS_WIND = 0  # Ground speed minus wind
+int8 SENSOR_1 = 1           # Sensor 1
+int8 SENSOR_2 = 2           # Sensor 2
+int8 SENSOR_3 = 3           # Sensor 3
+int8 SYNTHETIC = 4          # Synthetic airspeed 
```

Comment:
Can we give these a prefix? They are effectively and enum.

Reply:
SOURCE_*? Yes makes sense.